### ISSUE TITLE
METR bubble size labels

### ISSUE THREAD

**Issue by jdebacker** [open-source-economics/plots#32]
@brendancol, @MattHJensen made a good point that the bubble labels (for different amounts of assets determining the size of bubbles) are not correct.  I updated the asset data and fixed some of the problem, but I am not sure how to adjust the labels for the bubble size.  

Where would I look to adjust those?

**Comment by brendancol** [comment]
@jdebacker Bubble size labels change be changed with the `template.j2` files:

https://github.com/open-source-economics/plots/blob/master/contrib/corp_metr/template.j2#L240-L241

https://github.com/open-source-economics/plots/blob/master/contrib/corp_metr/template_portrait.j2#L239-L240

**Comment by jdebacker** [comment]
@brendancol  Thanks - I'll try to get in there tonight and edit those.  The `cut` to determine bubble size is always breaking into quartiles, right?  This despite the range specified in `build.py`:

```
SIZES = list(range(12, 30, 8))
df['size'] = pd.qcut(df['assets_c'].values, len(SIZES), labels=SIZES)
```

**Comment by brendancol** [comment]
@Jason, `qcut` cuts into `quantiles` based on number of labels provided

**Comment by jdebacker** [comment]
Fixed via PR #36.
